base/src/main/java/io/spine/string/FnStringifier.java

```diff
@@ -26,6 +26,9 @@
 
 /**
  * Abstract base for stringifiers that convert values using function objects.
+ *
+ * @param <T>
+ *         type of stringified objects
```

Comment:
@armiol, don't we need “the” before ”type” here and in other generic type descriptions in this PR?